packages/styles/scss/components/tag.scss

```diff
@@ -1,70 +1,70 @@
-@use '../config.scss' as c;
+@use 'sass:map';
+@use '../config.scss' as *;
+@use '../utilities/custom-properties.scss' as *;
+@use '../utilities/convert.scss';
+@use '../type/styles.scss' as type;
 
-@layer components {
-  // TODO: Finish tag component
-  .#{c.$prefix}-tag {
-    display: inline-flex;
-    align-items: center;
-    padding: 0.25rem 0.5rem;
-    font-size: var(--font-size-small, 0.875rem);
-    border-radius: 4px;
-    font-weight: var(--font-weight-medium, 500);
-    white-space: nowrap;
-    user-select: none;
-    cursor: default;
-    background-color: red;
-  }
 
-  /* Sizes */
-  .#{c.$prefix}-tag--small {
-    font-size: var(--font-size-xsmall, 0.75rem);
-    block-size: 1.75rem;
-    padding: 0 0.5rem;
-    border-radius: 4px;
-  }
+$sizes: (
+  "sm": (tagHeight: 1.125rem, xPadding: 0.5rem, crossPaddingRight: 1px, iconMargin: 0.25rem),
+  "md": (tagHeight: 1.5rem, xPadding: 0.5rem, crossPaddingRight: 0.25rem, iconMargin: 0.25rem),
+  "lg": (tagHeight: 2rem, xPadding: 0.75rem, crossPaddingRight: 0.5rem, iconMargin: 0.5rem)
+);
```

Comment:
Good use out of a map, try and use the $spacing tokens for margin and paddings though.